packages/api/shared/Route.ts

```diff
@@ -205,6 +203,8 @@ export default function createRoute<Params>(): RouteBuilder<Params> {
     },
     build() {
       return async (req: NextApiRequest, res: NextApiResponse) => {
+        await cors(req, res);
```

Comment:
We have to handle cors here so that it can provide a response for an OPTIONS request. Otherwise this will return 405 because there is no handler for it